### PR TITLE
Console command to move avatars to the avatar cache

### PR DESCRIPTION
**Pull request by annando**
To finalize the avatar file cache, there is now a command to create avatar cache files for avatars that are cached in the `photo` table.

The output may look slightly weird, but it is a process that will last several hours, maybe days. So there need to be some visible action there.

### CODE REVIEW

File: src/Console/MoveToAvatarCache.php [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+use Friendica\App\BaseURL;
+use Friendica\Contact\Avatar;
+use Friendica\Core\L10n;
+use Friendica\Model\Contact;
+use Friendica\Model\Photo;
+use Friendica\Util\Images;
+use Friendica\Object\Image;
+
+/**
+ * tool to control the list of ActivityPub relay servers from the CLI
```
Comment (tobiasd) [inline src/Console/MoveToAvatarCache.php]: This comment looks like a left over from the copy & paste ;-) could you update it?

Comment (annando) [reply src/Console/MoveToAvatarCache.php]: Done!

File: src/Console/MoveToAvatarCache.php [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+	protected function doExecute()
+	{
+		$condition = ["`avatar` != ? AND `photo` LIKE ? AND `uid` = ? AND `uri-id` != ? AND NOT `uri-id` IS NULL",
+			'', $this->baseurl->get() . '/photo/%', 0, 0];
+
+		$count    = 0;
+		$total    = $this->dba->count('contact', $condition);
+		$contacts = $this->dba->select('contact', ['id', 'avatar', 'photo', 'uri-id', 'url', 'avatar'], $condition, ['order' => ['id']]);
+		while ($contact = $this->dba->fetch($contacts)) {
+			echo ++$count . '/' . $total . "\t" . $contact['id'] . "\t" . $contact['url'] . "\t";
```
Comment (MrPetovan) [inline src/Console/MoveToAvatarCache.php]: Please use `$this->out()` for automatic line feed output.

**Review by MrPetovan — CHANGES_REQUESTED** [review]

File: src/Console/MoveToAvatarCache.php (hunk 2 above)
Comment (annando) [reply src/Console/MoveToAvatarCache.php]: Problem is that I need unbuffered and immediate output without a line feed.

Comment (annando) [reply src/Console/MoveToAvatarCache.php]: Update: The function can be called without a line feed, so I was able changing it.

**Review by MrPetovan — APPROVED** [review]

**Pull request merged**